src/interfaces/store-interfaces/state-interfaces/amalgamation-state-interface.ts

```diff
@@ -1,15 +1,7 @@
-import { AmalgamationTypes, ApprovalTypes, RelationshipTypes } from '@/enums'
-import { CourtOrderIF } from '@/interfaces'
+import { AmalgamationTypes } from '@/enums'
+import { BusinessIF } from '@/interfaces'
 
 export interface AmalgamationStateIF {
-  applicationDate?: string // YYYY-MM-DD
```

Comment:
Removed all the useless fields here for now. We'll eventually going to have to add lots of stuff in here depending on the ticket.